infra/Berksfile.lock

```diff
@@ -0,0 +1,71 @@
+DEPENDENCIES
+  apt
+  cassandra
+  cassandra-dse (~> 4.3.0)
+  docker
+
+GRAPH
+  apt (2.9.2)
+  ark (3.1.0)
+    build-essential (>= 0.0.0)
+    seven_zip (>= 0.0.0)
+    windows (>= 0.0.0)
+  boost (2.1.0)
+    build-essential (>= 0.0.0)
+  build-essential (8.0.3)
+    mingw (>= 1.1)
+    seven_zip (>= 0.0.0)
+  cassandra (0.2.4)
+    apt (>= 0.0.0)
+    install_from (>= 0.0.0)
+    iptables (>= 0.0.0)
+    java (>= 0.0.0)
+    metachef (>= 0.0.0)
+    runit (>= 0.0.0)
+    thrift (>= 0.0.0)
+    volumes (>= 0.0.0)
+  cassandra-dse (4.3.0)
+    apt (>= 0.0.0)
+    ark (>= 0.0.0)
+    java (>= 0.0.0)
+    systemd (>= 0.0.0)
+    ulimit (>= 0.0.0)
+    yum (>= 0.0.0)
+  compat_resource (12.5.14)
+  docker (2.2.10)
+    compat_resource (>= 12.5.14)
+  homebrew (4.2.0)
+  install_from (3.0.4)
+  iptables (4.2.0)
+  java (1.50.0)
+    apt (>= 0.0.0)
+    homebrew (>= 0.0.0)
+    windows (>= 0.0.0)
+  metachef (3.0.4)
+  mingw (2.0.1)
+    seven_zip (>= 0.0.0)
+  ohai (5.1.0)
+  packagecloud (0.3.0)
+  python (1.4.6)
+    build-essential (>= 0.0.0)
+    yum-epel (>= 0.0.0)
+  runit (3.0.5)
+    packagecloud (>= 0.0.0)
+    yum-epel (>= 0.0.0)
+  seven_zip (2.0.2)
+    windows (>= 1.2.2)
+  systemd (2.1.3)
+  thrift (1.2.0)
+    boost (>= 0.0.0)
+    build-essential (>= 0.0.0)
+    python (>= 0.0.0)
+  ulimit (0.4.0)
+  volumes (3.0.4)
+    metachef (>= 0.0.0)
+    xfs (>= 0.0.0)
+  windows (3.1.0)
+    ohai (>= 4.0.0)
+  xfs (2.0.1)
+  yum (5.0.1)
+  yum-epel (1.0.2)
+    yum (>= 3.6)
```

Comment:
this is a lock file, so everyone has the same versions of things, one does not edit it by hand